### ISSUE TITLE
Bulk Late Days Table Scroll Overlapping Columns

### ISSUE THREAD

**Issue by RitaLei123** [Submitty/Submitty#9885]
[](url)**Describe the bug**
The Bulk Late Days table columns overlap with each other when the user scrolls right.

**Expected behavior**
When you scroll right in the table in Bulk Late Days, the table columns: user ID, first name, last name, initial, late allowed homework column overlap with all the columns.

**To Reproduce**
1. Login using instructor and go to the sample course.
2. Click on the Bulk Late Days category. 
3. Keep on scrolling right in the table.

**Configuration**
 - Windows
 - Chrome

**Screenshots**
<img width="943" alt="image" src="https://github.com/Submitty/Submitty/assets/117528498/6141d950-a7ec-4fbc-ab09-c27f3532e8c8">

**Additional context**
This is a table issue. Here is a clip:
https://github.com/Submitty/Submitty/assets/117528498/6e823f69-2ccd-4cf0-a4e5-e23ae1b62fae

**Comment by kanugurajesh** [comment]
hi @RitaLei123 can you assign me this issue ?

**Comment by Youssef-Chip** [comment]
Since there are many columns in the table, I made the design like that so it could be easier to navigate, but we should have a discussion during the meeting to see if we need to revise it.

**Comment by bmcutler** [comment]
Hi @kanugurajesh 

Thank you for your interest in the Submitty open source project.
We welcome contributions from new developers!
However we do not use the Github issue 'assign' feature for first time prospective contributors.

Please read our documentation on how to to get started with Submitty:
https://submitty.org/developer/getting_started/index
Specifically in setting up your development environment:
https://submitty.org/developer/getting_started/vm_install_using_vagrant
And making a pull request:
https://submitty.org/developer/getting_started/make_a_pull_request
You do not need to be assigned to an issue to create a pull request that will be reviewed by our team and then merged if it appropriately resolves the issue.
Also join our Slack server to discuss technical questions:
https://submitty.org/index/contact
